spacemacs/packages.el

```diff
@@ -1677,14 +1677,19 @@ If ARG is non nil then `ag' and `pt' and ignored."
           (call-interactively (spacemacs//helm-projectile-do-search-find-tool
                                tools))))
 
+      (setq helm-projectile-sources-list '(helm-source-projectile-projects
+                                           helm-source-projectile-files-list
+                                           helm-source-recentf
+                                           helm-source-pp-bookmarks
+                                           helm-source-file-cache))
```

Comment:
You can read about [File Name Cache](http://www.gnu.org/software/emacs/manual/html_node/emacs/File-Name-Cache.html).

To try it out:

- `M-x file-cache-add-directory` and select a directory.
- If you applied this PR, run `C-/`.
- See the file cache candidates: files in the above directory.